docs/concepts/glossary.md

```diff
@@ -0,0 +1,23 @@
+# Glossary
+
+Most terms used in Beaker documentation should be common. The following documents project jargon which may otherwise be confusing.
+
+## Coordinator
+
+The Coordinator is the system on which Beaker itself is run. In many environments this will be a local host, typically a developer's primary machine. Used instead of [Master](#master) to avoid confusion.
+
+## Hypervisor
+
+The Hypervisor component manages the hypervisor adapters that connect to hypervisor systems (e.g. ESXi, Cisco, VMPooler) that provide SUTs.
+
+## Master
+
+A SUT running as a Puppet Master.
+
+## SUT
+
+(See [System Under Test](#system-under-test).)
```

Comment:
This shouldn't have its own section. Since it's a shortening of System Under Test, it should go in that section.